README.md

```diff
@@ -183,17 +183,26 @@ HelmChart(k8s_deployment_stack, "myapp", cluster=cluster, namespace="myapp-names
 
 ### Kubernetes / AWS EKS
 
-`ca_cdk_constructs` includes a vendored cdk8s library which can be imported with:
+`ca_cdk_constructs` includes a vendored cdk8s library.
```

Comment:
You need to do it if your project imports the vendored library from our cdk constructs code AND we have upgraded the eks version.

Basically, by importing it you are making sure that you target the current k8s version.

It might have been better to put the cdk8s code in a separate python package...

Reply:
Thanks - could we add this to the readme then, so it has what you need to do and when you'd need to do it?